cmd/lint.go

```diff
@@ -44,12 +43,17 @@ func lintCommandAction(cmd *cobra.Command, args []string) error {
 		return errors.Wrap(err, "locating package root failed")
 	}
 
-	ok, err := docs.IsReadmeUpToDate()
+	errNames, notOKNames, err := docs.AreReadmesUpToDate()
 	if err != nil {
-		return errors.Wrapf(err, "can't check if %s file is up-to-date", docs.ReadmeFile)
+		return errors.Wrap(err, "can't check if readme files are up-to-date")
 	}
-	if !ok {
-		return fmt.Errorf("%s file is outdated. Rebuild the package with 'elastic-package build'", docs.ReadmeFile)
+
+	if errNames != "" {
+		return errors.Wrapf(err, "can't check if %sare up-to-date", errNames)
```

Comment:
nit: space `%sare`

Reply:
I intentionally removed the space because the errNames string already is ending with an empty space :) But I'm adding one here too so the code is more readable.